contrib/building.md

```diff
@@ -258,6 +259,38 @@ copy .\packages\openssl-windows_x64-windows-static\lib\libeay32.lib C:\src\vcpkg
 copy .\packages\openssl-windows_x64-windows-static\lib\ssleay32.lib C:\src\vcpkg\installed\x64-windows-static\lib
 ```
 
+#### Building for ARM/Linux
+
+You can build Realm JS for ARM/Linux from source and include it in your own project.
+
+The following instructions assume you are using [Debian GNU/Linux](https://www.debian.org) or a derived distribution.
+
+First you need to have your build environment set up:
+
+```sh
+apt install build-essential cmake git openssl-dev
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+nvm install 16 # you can use any supported node version
+```
+
+You can now build Realm JS from source:
+
+```sh 
+export REALM_USE_SYSTEM_OPENSSL=1
+git clone https://github.com/realm/realm-js
+cd realm-js
+git submodule update —-init —-recursive
+npm install
```

Comment:
We should probably remove the call to `cmake-js` from our `install` script in the future, because of the confusion it causes when prebuilt binaries are missing and not installing from source like this.

I'd suggest splitting this into install and an explicit build command instead:

```suggestion
npm install --ignore-scripts
npx cmake-js build
```